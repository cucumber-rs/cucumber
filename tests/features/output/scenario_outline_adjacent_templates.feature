Feature: Outline

  Scenario Outline: foo
    Given foo is <bar1><bar1>
    When foo is <bar2><bar1><bar2>
    Then foo is <bar3><bar2>

    Examples:
      | bar1 | bar2 | bar3 |
      |  0   |  1   |  2   |
