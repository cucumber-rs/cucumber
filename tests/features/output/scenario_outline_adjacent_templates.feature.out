Started
Feature(Feature { keyword: "Feature", name: "Outline", description: None, background: None, scenarios: [Scenario { keyword: "Scenario Outline", name: "foo", steps: [Step { keyword: "Given", ty: Given, value: "foo is 00", docstring: None, table: None, position: LineCol { line: 4, col: 5 } }, Step { keyword: "When", ty: When, value: "foo is 101", docstring: None, table: None, position: LineCol { line: 5, col: 5 } }, Step { keyword: "Then", ty: Then, value: "foo is 21", docstring: None, table: None, position: LineCol { line: 6, col: 5 } }], examples: [Examples { keyword: "Examples", table: Table { rows: [["bar1", "bar2", "bar3"], ["0", "1", "2"]], position: LineCol { line: 9, col: 7 } }, tags: [], position: LineCol { line: 8, col: 5 } }], tags: [], position: LineCol { line: 9, col: 5 } }], rules: [], tags: [], position: LineCol { line: 1, col: 1 }, }, Started)
Feature(Feature { keyword: "Feature", name: "Outline", description: None, background: None, scenarios: [Scenario { keyword: "Scenario Outline", name: "foo", steps: [Step { keyword: "Given", ty: Given, value: "foo is 00", docstring: None, table: None, position: LineCol { line: 4, col: 5 } }, Step { keyword: "When", ty: When, value: "foo is 101", docstring: None, table: None, position: LineCol { line: 5, col: 5 } }, Step { keyword: "Then", ty: Then, value: "foo is 21", docstring: None, table: None, position: LineCol { line: 6, col: 5 } }], examples: [Examples { keyword: "Examples", table: Table { rows: [["bar1", "bar2", "bar3"], ["0", "1", "2"]], position: LineCol { line: 9, col: 7 } }, tags: [], position: LineCol { line: 8, col: 5 } }], tags: [], position: LineCol { line: 9, col: 5 } }], rules: [], tags: [], position: LineCol { line: 1, col: 1 }, }, Scenario(Scenario { keyword: "Scenario Outline", name: "foo", steps: [Step { keyword: "Given", ty: Given, value: "foo is 00", docstring: None, table: None, position: LineCol { line: 4, col: 5 } }, Step { keyword: "When", ty: When, value: "foo is 101", docstring: None, table: None, position: LineCol { line: 5, col: 5 } }, Step { keyword: "Then", ty: Then, value: "foo is 21", docstring: None, table: None, position: LineCol { line: 6, col: 5 } }], examples: [Examples { keyword: "Examples", table: Table { rows: [["bar1", "bar2", "bar3"], ["0", "1", "2"]], position: LineCol { line: 9, col: 7 } }, tags: [], position: LineCol { line: 8, col: 5 } }], tags: [], position: LineCol { line: 9, col: 5 } }, Started))
Feature(Feature { keyword: "Feature", name: "Outline", description: None, background: None, scenarios: [Scenario { keyword: "Scenario Outline", name: "foo", steps: [Step { keyword: "Given", ty: Given, value: "foo is 00", docstring: None, table: None, position: LineCol { line: 4, col: 5 } }, Step { keyword: "When", ty: When, value: "foo is 101", docstring: None, table: None, position: LineCol { line: 5, col: 5 } }, Step { keyword: "Then", ty: Then, value: "foo is 21", docstring: None, table: None, position: LineCol { line: 6, col: 5 } }], examples: [Examples { keyword: "Examples", table: Table { rows: [["bar1", "bar2", "bar3"], ["0", "1", "2"]], position: LineCol { line: 9, col: 7 } }, tags: [], position: LineCol { line: 8, col: 5 } }], tags: [], position: LineCol { line: 9, col: 5 } }], rules: [], tags: [], position: LineCol { line: 1, col: 1 }, }, Scenario(Scenario { keyword: "Scenario Outline", name: "foo", steps: [Step { keyword: "Given", ty: Given, value: "foo is 00", docstring: None, table: None, position: LineCol { line: 4, col: 5 } }, Step { keyword: "When", ty: When, value: "foo is 101", docstring: None, table: None, position: LineCol { line: 5, col: 5 } }, Step { keyword: "Then", ty: Then, value: "foo is 21", docstring: None, table: None, position: LineCol { line: 6, col: 5 } }], examples: [Examples { keyword: "Examples", table: Table { rows: [["bar1", "bar2", "bar3"], ["0", "1", "2"]], position: LineCol { line: 9, col: 7 } }, tags: [], position: LineCol { line: 8, col: 5 } }], tags: [], position: LineCol { line: 9, col: 5 } }, Step(Step { keyword: "Given", ty: Given, value: "foo is 00", docstring: None, table: None, position: LineCol { line: 4, col: 5 } }, Started)))
Feature(Feature { keyword: "Feature", name: "Outline", description: None, background: None, scenarios: [Scenario { keyword: "Scenario Outline", name: "foo", steps: [Step { keyword: "Given", ty: Given, value: "foo is 00", docstring: None, table: None, position: LineCol { line: 4, col: 5 } }, Step { keyword: "When", ty: When, value: "foo is 101", docstring: None, table: None, position: LineCol { line: 5, col: 5 } }, Step { keyword: "Then", ty: Then, value: "foo is 21", docstring: None, table: None, position: LineCol { line: 6, col: 5 } }], examples: [Examples { keyword: "Examples", table: Table { rows: [["bar1", "bar2", "bar3"], ["0", "1", "2"]], position: LineCol { line: 9, col: 7 } }, tags: [], position: LineCol { line: 8, col: 5 } }], tags: [], position: LineCol { line: 9, col: 5 } }], rules: [], tags: [], position: LineCol { line: 1, col: 1 }, }, Scenario(Scenario { keyword: "Scenario Outline", name: "foo", steps: [Step { keyword: "Given", ty: Given, value: "foo is 00", docstring: None, table: None, position: LineCol { line: 4, col: 5 } }, Step { keyword: "When", ty: When, value: "foo is 101", docstring: None, table: None, position: LineCol { line: 5, col: 5 } }, Step { keyword: "Then", ty: Then, value: "foo is 21", docstring: None, table: None, position: LineCol { line: 6, col: 5 } }], examples: [Examples { keyword: "Examples", table: Table { rows: [["bar1", "bar2", "bar3"], ["0", "1", "2"]], position: LineCol { line: 9, col: 7 } }, tags: [], position: LineCol { line: 8, col: 5 } }], tags: [], position: LineCol { line: 9, col: 5 } }, Step(Step { keyword: "Given", ty: Given, value: "foo is 00", docstring: None, table: None, position: LineCol { line: 4, col: 5 } }, Passed(CaptureLocations(Locations([Some(0), Some(9), Some(7), Some(9)]))))))
Feature(Feature { keyword: "Feature", name: "Outline", description: None, background: None, scenarios: [Scenario { keyword: "Scenario Outline", name: "foo", steps: [Step { keyword: "Given", ty: Given, value: "foo is 00", docstring: None, table: None, position: LineCol { line: 4, col: 5 } }, Step { keyword: "When", ty: When, value: "foo is 101", docstring: None, table: None, position: LineCol { line: 5, col: 5 } }, Step { keyword: "Then", ty: Then, value: "foo is 21", docstring: None, table: None, position: LineCol { line: 6, col: 5 } }], examples: [Examples { keyword: "Examples", table: Table { rows: [["bar1", "bar2", "bar3"], ["0", "1", "2"]], position: LineCol { line: 9, col: 7 } }, tags: [], position: LineCol { line: 8, col: 5 } }], tags: [], position: LineCol { line: 9, col: 5 } }], rules: [], tags: [], position: LineCol { line: 1, col: 1 }, }, Scenario(Scenario { keyword: "Scenario Outline", name: "foo", steps: [Step { keyword: "Given", ty: Given, value: "foo is 00", docstring: None, table: None, position: LineCol { line: 4, col: 5 } }, Step { keyword: "When", ty: When, value: "foo is 101", docstring: None, table: None, position: LineCol { line: 5, col: 5 } }, Step { keyword: "Then", ty: Then, value: "foo is 21", docstring: None, table: None, position: LineCol { line: 6, col: 5 } }], examples: [Examples { keyword: "Examples", table: Table { rows: [["bar1", "bar2", "bar3"], ["0", "1", "2"]], position: LineCol { line: 9, col: 7 } }, tags: [], position: LineCol { line: 8, col: 5 } }], tags: [], position: LineCol { line: 9, col: 5 } }, Step(Step { keyword: "When", ty: When, value: "foo is 101", docstring: None, table: None, position: LineCol { line: 5, col: 5 } }, Started)))
Feature(Feature { keyword: "Feature", name: "Outline", description: None, background: None, scenarios: [Scenario { keyword: "Scenario Outline", name: "foo", steps: [Step { keyword: "Given", ty: Given, value: "foo is 00", docstring: None, table: None, position: LineCol { line: 4, col: 5 } }, Step { keyword: "When", ty: When, value: "foo is 101", docstring: None, table: None, position: LineCol { line: 5, col: 5 } }, Step { keyword: "Then", ty: Then, value: "foo is 21", docstring: None, table: None, position: LineCol { line: 6, col: 5 } }], examples: [Examples { keyword: "Examples", table: Table { rows: [["bar1", "bar2", "bar3"], ["0", "1", "2"]], position: LineCol { line: 9, col: 7 } }, tags: [], position: LineCol { line: 8, col: 5 } }], tags: [], position: LineCol { line: 9, col: 5 } }], rules: [], tags: [], position: LineCol { line: 1, col: 1 }, }, Scenario(Scenario { keyword: "Scenario Outline", name: "foo", steps: [Step { keyword: "Given", ty: Given, value: "foo is 00", docstring: None, table: None, position: LineCol { line: 4, col: 5 } }, Step { keyword: "When", ty: When, value: "foo is 101", docstring: None, table: None, position: LineCol { line: 5, col: 5 } }, Step { keyword: "Then", ty: Then, value: "foo is 21", docstring: None, table: None, position: LineCol { line: 6, col: 5 } }], examples: [Examples { keyword: "Examples", table: Table { rows: [["bar1", "bar2", "bar3"], ["0", "1", "2"]], position: LineCol { line: 9, col: 7 } }, tags: [], position: LineCol { line: 8, col: 5 } }], tags: [], position: LineCol { line: 9, col: 5 } }, Step(Step { keyword: "When", ty: When, value: "foo is 101", docstring: None, table: None, position: LineCol { line: 5, col: 5 } }, Failed(Some(CaptureLocations(Locations([Some(0), Some(10), Some(7), Some(10)]))), Some(World(1)), Panic(Any { .. })))))
Feature(Feature { keyword: "Feature", name: "Outline", description: None, background: None, scenarios: [Scenario { keyword: "Scenario Outline", name: "foo", steps: [Step { keyword: "Given", ty: Given, value: "foo is 00", docstring: None, table: None, position: LineCol { line: 4, col: 5 } }, Step { keyword: "When", ty: When, value: "foo is 101", docstring: None, table: None, position: LineCol { line: 5, col: 5 } }, Step { keyword: "Then", ty: Then, value: "foo is 21", docstring: None, table: None, position: LineCol { line: 6, col: 5 } }], examples: [Examples { keyword: "Examples", table: Table { rows: [["bar1", "bar2", "bar3"], ["0", "1", "2"]], position: LineCol { line: 9, col: 7 } }, tags: [], position: LineCol { line: 8, col: 5 } }], tags: [], position: LineCol { line: 9, col: 5 } }], rules: [], tags: [], position: LineCol { line: 1, col: 1 }, }, Scenario(Scenario { keyword: "Scenario Outline", name: "foo", steps: [Step { keyword: "Given", ty: Given, value: "foo is 00", docstring: None, table: None, position: LineCol { line: 4, col: 5 } }, Step { keyword: "When", ty: When, value: "foo is 101", docstring: None, table: None, position: LineCol { line: 5, col: 5 } }, Step { keyword: "Then", ty: Then, value: "foo is 21", docstring: None, table: None, position: LineCol { line: 6, col: 5 } }], examples: [Examples { keyword: "Examples", table: Table { rows: [["bar1", "bar2", "bar3"], ["0", "1", "2"]], position: LineCol { line: 9, col: 7 } }, tags: [], position: LineCol { line: 8, col: 5 } }], tags: [], position: LineCol { line: 9, col: 5 } }, Finished))
Feature(Feature { keyword: "Feature", name: "Outline", description: None, background: None, scenarios: [Scenario { keyword: "Scenario Outline", name: "foo", steps: [Step { keyword: "Given", ty: Given, value: "foo is 00", docstring: None, table: None, position: LineCol { line: 4, col: 5 } }, Step { keyword: "When", ty: When, value: "foo is 101", docstring: None, table: None, position: LineCol { line: 5, col: 5 } }, Step { keyword: "Then", ty: Then, value: "foo is 21", docstring: None, table: None, position: LineCol { line: 6, col: 5 } }], examples: [Examples { keyword: "Examples", table: Table { rows: [["bar1", "bar2", "bar3"], ["0", "1", "2"]], position: LineCol { line: 9, col: 7 } }, tags: [], position: LineCol { line: 8, col: 5 } }], tags: [], position: LineCol { line: 9, col: 5 } }], rules: [], tags: [], position: LineCol { line: 1, col: 1 }, }, Finished)
Finished
